content/index.fr.md

```diff
@@ -78,12 +78,12 @@ Les étapes importantes par lesquelles doit passer le "rapport technique" pour d
-3.  {:#cr}![using computer with technical report:]({{ "/content-images/wai-about-w3c-process/cr.gif" | relative_url }}){:style="float:left; margin-right: 1em; clear: left;"} **Recommandation candidate** : L'objectif premier du statut de recommandation candidate est de s'assurer que le rapport technique peut être implémenté. Le W3C encourage les développeurs à utiliser le rapport technique dans leurs projets. Le rapport technique est stable à cette étape ; toutefois, il pourrait être modifié en fonction des résultats de l'implémentation.
-4.  {:#pr}![endorsing technical report:]({{ "/content-images/wai-about-w3c-process/pr.gif" | relative_url }}){:style="float:left; margin-right: 1em; clear: left;"} **Proposition de recommandation** : après l'implémentation de chaque étape du rapport technique, le W3C annonce la proposition de recommandation. À ce stade, le rapport est soumis aux membres du W3C pour être approuvé.
+3.  {:#cr}![using computer with technical report:]({{ "/content-images/wai-about-w3c-process/cr.gif" | relative_url }}){:style="float:left; margin-right: 1em; clear: left;"} **Recommandation candidate** : L'objectif principal du statut de recommandation candidate est de s'assurer que le rapport technique peut être implémenté. Le W3C encourage les développeurs à utiliser le rapport technique dans leurs projets. Le rapport technique est stable à cette étape ; toutefois, il pourrait être modifié en fonction des résultats de l'implémentation.
+4.  {:#pr}![endorsing technical report:]({{ "/content-images/wai-about-w3c-process/pr.gif" | relative_url }}){:style="float:left; margin-right: 1em; clear: left;"} **Proposition de recommandation** : après l'implémentation de chaque fonctionnalité du rapport technique, le W3C annonce la proposition de recommandation. À ce stade, le rapport est soumis aux membres du W3C pour être approuvé.
 5.  {:#rec}![approved technical report:]({{ "/content-images/wai-about-w3c-process/rec.gif" | relative_url }}){:style="float:left; margin-right: 1em; clear: left;"} **Recommandation du W3C
     (Standard Web)** : Lorsque le rapport technique est approuvé par les membres, le directeur et le public du W3C, il est publié en tant que recommandation. Le W3C encourage le déploiement à grande échelle de ses recommandations.
 
-Il s'agit d'une description simplifiée du processus. Pour la version définitive, consultez le [document sur le processus du W3C, section 7 : Processus de développement d'un rapport technique du W3C](http://www.w3.org/Consortium/Process/#Reports).
+C'était une description simplifiée du processus. Pour la version définitive, consultez le [document sur le processus du W3C, section 7 : Processus de développement d'un rapport technique du W3C](http://www.w3.org/Consortium/Process/#Reports).
```

Comment:
@vmmiller I tend to prefer "Il s'agit d'une description simplifiée du processus", that conveys the message "You have just read a simplified description of the process". That is what I understand when I read "That was a simplified description of the process".

In that context, "C'était une description simplifiée du process" sounds a bit too informal. Don't you think ?